### PR TITLE
Upload pre-trained weights for MobileNet and ResNeXt

### PR DESCRIPTION
**Pull request by fmassa**
Also move weights from ShuffleNet to PyTorch bucket.

I also renamed `shufflenetv2` to `shufflenet_v2` to make it consistent with the other models. This is fine because there hasn't been a release with `shufflenet_v2` yet.

### CODE REVIEW

**Comment by barrh** [comment]
@fmassa are there plans for torchvision v0.3?

**Comment by fmassa** [comment]
@barrh yes, today :-)
